checkov/common/output/report.py

```diff
@@ -242,24 +244,29 @@ def print_console(
             else:
                 message = f"\nPassed checks: {summary['passed']}, Failed checks: {summary['failed']}, Skipped checks: {summary['skipped']}\n"
         print(colored(message, "cyan"))
-
+        output_data += message
```

Comment:
can you print the `output_data` in the end? this way you won't need to print and add to this variable each line